### ISSUE TITLE
Refine style to centralize image

### ISSUE THREAD

**Issue by swallowchen1359** [WishCanTaiwan/VocabuLazy#56]
Refine styles.xml to centralize image and fix some phones (api 5.0.2) will be with wrong background color.

**Comment by ylallencheng** [comment]
@swallowchen1359 I believe this issue had been solved by using dp to set up text size?

**Comment by swallowchen1359** [comment]
@allencheng07 NO, using dp only fixes the text size issue. But background colour is not fixed yet. And I haven't checked which images are not centralized.
CANNOT CLOSE YET

**Comment by ylallencheng** [comment]
No problem!
